.changeset/search-blankly-have-a-nice-life.md

```diff
@@ -0,0 +1,14 @@
+---
+'@backstage/plugin-search-backend-node': minor
+'@backstage/search-common': minor
+---
+
+The Backstage Search Platform's indexing process has been rewritten as a stream
```

Comment:
We've started marking breaking changes with [`**BREAKING**: `](https://github.com/backstage/backstage/blob/1e57d1c23e5cc236796d7166aaf7b84c86e9b6e0/docs/getting-started/contributors.md#breaking-changes-not-caught-by-the-type-checker-should-be-clearly-marked-with-bold-breaking-text) so that they're a bit easier to spot